docs/design/two-phase-transfers.md

```diff
@@ -24,28 +24,24 @@ increased on the relevant accounts at this point.
 * [`post_pending_transfer`](../reference/transfers.md#flagspost_pending_transfer): Move all (or part) of the reserved funds to the pending transfer's destination.
 * [`void_pending_transfer`](../reference/transfers.md#flagsvoid_pending_transfer): Revert all of the reserved funds to the original account.
 
-A pending transfer can only be posted or voided once. It cannot be
-posted twice or voided then posted, etc.
+## Resolving a two-phase transfer
 
 When the pending transfer is resolved (posted or voided), the
```

Comment:
Just noticed this, but we should define the term "resolved" in the context of two-phase transfers. (Unless we do already and I missed it).

Reply:
Hmm what do you mean? It's defined this sentence as being posted or voided.

Reply:
Ignore me, not sure what I was thinking. :facepalm: